docs/installation/amd/installation.md

```diff
@@ -0,0 +1,134 @@
+# AMD Installation
+
+## Supported
+
+- **Target Devices**
+    + [x] AMD GPUs
+- **Operating Systems**
+    + [x] Linux AMD64
+    + [x] Linux ARM64
+- **Available Inference Engines**
```

Comment:
Call it Inference Backends to align with the UI and other parts of the documentation.